packages/next/test/fixtures/00-app-dir-segment-prefetch-incremental-conflicting-routes/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "next": "https://vercel-packages.vercel.app/next/commits/2fe49e013be98130279dd13b8199afcde23f6f56/next",
```

Comment:
I'll change this to "latest" before landing. Depends on changes in https://github.com/vercel/next.js/pull/78568.